### ISSUE TITLE
Sparce NCA and Zelda TOTK

### ISSUE THREAD

**Issue by Guigs99** [Huntereb/Awoo-Installer#191]
-  Is anyone having trouble with Games with Sparce NCA (No More Heroes 3 or Splatoon 3, etc) ? Did any other game gave you trouble/how did you fix it? 

- On another note, is Awoo capable of installing Tears of The Kingdom ? Even on latest atmosphere and sigpatches

**Comment by remlei** [comment]
yeah same here, mine is a xci format of zelda totk and its a untrimmed 32gb image.

it stubbornsly failing to install, tried trimming the image xci dump and that seems to fix the issue.

I guess awoo and/or NS-USBloader cant really handle those untrimmed xci files.
